schemas/CreateAccountInput.json

```
@@ -0,0 +1,271 @@
+          "example": "2016-11-27T13:19:56+00:00"
+        },
+        "product_id": {
+          "type": "string",
+          "description": "A unique product ID generated by Canopy.",
+          "example": "31mNprzLd2bKl6koVna68ARM"
+        },
+        "external_product_id": {
+          "type": "string",
+          "description": "A unique external ID that may be used interchangeably with the Canopy-generated product ID. Either this, or the Canopy product ID must be passed at the time of account creation.",
+          "example": "95fea111-d121-40f8-9e8e-a6c335e48a45"
+        },
+        "external_fields": {
+          "type": "array",
+          "nullable": true,
+          "maxItems": 10,
+          "description": "External fields can be used to relate accounts created in Canopy to entities in your or any external system.",
+          "items": {
+            "type": "object",
+            "properties": {
+              "key": {
+                "type": "string",
+                "description": "key: i.e. Name of the External Party",
+                "example": "Umbrella Corporation"
+              },
+              "value": {
+                "type": "string",
+                "description": "value: i.e. External Account ID",
+                "example": "22445702-a389-431f-927d-07b8d0750787"
+              }
+            }
+          }
+        },
+        "payment_processor_config": {
+          "type": "object",
+          "properties": {
+            "payment_processor_name": {
+              "type": "string",
+              "enum": ["NONE", "REPAY"],
+              "description": "Indicates the active payment processor used for payments made by the account. If `NONE`, Canopy will not trigger payments to any payment processor when they occur.",
+              "example": "REPAY",
+              "default": "NONE"
+            },
+            "autopay_enabled": {
+              "type": "boolean",
+              "description": "Indicates whether autopay is enabled for this account. Autopay is triggered one day prior to due date. If `payment_processor_name` is `NONE`, autopay will not be triggered for the account regardless of this field.",
+              "example": true,
+              "default": false
+            },
+            "repay_config": {
+              "type": "object",
+              "description": "Sensitive bank information will be stored as a secured token for payments in place of the raw account details.",
+              "required": [
+                "repay_payment_method",
+                "repay_check_type",
+                "repay_account_type",
+                "repay_transit_number",
+                "repay_account_number",
+                "repay_name_on_check"
+              ],
+              "properties": {
+                "repay_payment_method": {
+                  "type": "string",
+                  "enum": ["ACH"],
+                  "description": "The payment method to use for this account"
```

```suggestion
                  "description": "The payment method used by the account"
```

This is a debatable suggestion. I'd like to change our verbiage to move away from "for this" to "by" to imply that the account is taking action based on the configurations.